crates/ruma-events/src/tag.rs

```diff
@@ -77,7 +78,7 @@ impl Error for InvalidUserTagName {}
 #[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord)]
 #[cfg_attr(not(feature = "unstable-exhaustive-types"), non_exhaustive)]
 pub enum TagName {
-    /// `m.favourite`: The user's favourite rooms. These should be shown with higher precedence
+    /// `m.favorite`: The user's favorite rooms. These should be shown with higher precedence
```

Comment:
It's actually called `m.favourite`, though it makes sense to change the second occurrence on the line since we also use the American spelling for the variant's name.

Reply:
This is an unrelated change though and should be in a separate PR.